packages/block-library/src/embed/edit.js

```diff
@@ -186,14 +186,16 @@ const EmbedEdit = ( props ) => {
 		);
 	}
 
-	// translators: %s: type of embed e.g: "YouTube", "Twitter", etc. "Embed" is used when no specific type exists
-	const label = sprintf( __( '%s URL' ), title );
-
+	const label = Platform.select( {
+		// translators: %s: type of embed e.g: "YouTube", "Twitter", etc. "Embed" is used when no specific type exists
+		web: sprintf( __( '%s URL' ), title ),
+		native: title,
+	} );
 	// No preview, or we can't embed the current URL, or we've clicked the edit button.
 	const showEmbedPlaceholder = ! preview || cannotEmbed || isEditingURL;
 	if ( showEmbedPlaceholder ) {
 		return (
-			<View { ...blockProps }>
+			<View style={ { flex: 1 } } { ...blockProps }>
```

Comment:
> Thanks @ceyhun I resolved this issue by adding flex : 1 to the View e24a0c6 let me know if this is approach is appropriate given the fact that the change is in a file shared with web/mobile.

Seems like there's a native version of the `useBlockProps` hook that's returning the `blockProps`. Do you think it would be fine to maybe add it there, so every other `View` in this file gets it? That could also pass it to some other native components where the hook is used, but I guess we can check if that's an issue or not.

Reply:
Thanks for mentioning this. I did an implementation of this here 76c1c94 Also the placeholder was not focusing so I added that as well f792403 Let me know if my usage of the spread operators is fine.

Reply:
Changes look good to me 👍 To check any affected native blocks that are using `useBlockProps` already, I searched for `useBlockProps` and looked into blocks available on mobile that are not overriding the style attribute of `useBlockProps`. 
I think these are the ones that could be affected by this change:

- Gallery
- List
- Preformatted
- Pullquote
- Search
- Spacer
- Verse

I wanted to test those blocks, but I got a BlockAlignmentUI crash, which should be fixed by [this PR](https://github.com/WordPress/gutenberg/pull/31581).

`useBlockProps` is also used in a couple of `save.js` files, but only the `useBlockProps.save` method seems to get called in those files, so I think they should not be affected by this change.

Reply:
Thanks for sharing this @ceyhun since the breakage was fixed, I will be doing a test of these blocks to ensure no regressions have taken place as a result of these changes.